inject.sh

```diff
@@ -1,135 +1,140 @@
+ fi
 if [ "$APP_DEBUG" != '' ]; then
-    sed -i "s|APP_DEBUG=.*|APP_DEBUG=${APP_DEBUG}|i" /conf/.env
-fi
+    replace_or_insert "APP_DEBUG" "$APP_DEBUG"
+ fi
 if [ "$APP_URL" != '' ]; then
-    sed -i "s|APP_URL=.*|APP_URL=${APP_URL}|i" /conf/.env
-fi
+    replace_or_insert "APP_URL" "$APP_URL"
+ fi
 if [ "$APP_FORCE_HTTPS" != '' ]; then
-    sed -i "s|APP_FORCE_HTTPS=.*|APP_FORCE_HTTPS=${APP_FORCE_HTTPS}|i" /conf/.env
-fi
+    replace_or_insert "APP_FORCE_HTTPS" "$APP_FORCE_HTTPS"
+ fi
 if [ "$APP_DIR" != '' ]; then
-    sed -i "s|APP_DIR=.*|APP_DIR=${APP_DIR}|i" /conf/.env
-fi
+    replace_or_insert "APP_DIR" "$APP_DIR"
+ fi
 if [ "$DEBUGBAR_ENABLED" != '' ]; then
-    sed -i "s|DEBUGBAR_ENABLED=.*|DEBUGBAR_ENABLED=${DEBUGBAR_ENABLED}|i" /conf/.env
-fi
+    replace_or_insert "DEBUGBAR_ENABLED" "$DEBUGBAR_ENABLED"
+ fi
 if [ "$LEGACY_V4_REDIRECT" != '' ]; then
-    sed -i "s|LEGACY_V4_REDIRECT=.*|LEGACY_V4_REDIRECT=${LEGACY_V4_REDIRECT}|i" /conf/.env
-fi
+    replace_or_insert "LEGACY_V4_REDIRECT" "$LEGACY_V4_REDIRECT"
+ fi
 if [ "$DB_OLD_LYCHEE_PREFIX" != '' ]; then
-    sed -i "s|DB_OLD_LYCHEE_PREFIX=.*|DB_OLD_LYCHEE_PREFIX=${DB_OLD_LYCHEE_PREFIX}|i" /conf/.env
-fi
+    replace_or_insert "DB_OLD_LYCHEE_PREFIX" "$DB_OLD_LYCHEE_PREFIX"
+ fi
 if [ "$DB_CONNECTION" != '' ]; then
-    sed -i "s|DB_CONNECTION=.*|DB_CONNECTION=${DB_CONNECTION}|i" /conf/.env
-fi
+    replace_or_insert "DB_CONNECTION" "$DB_CONNECTION"
+ fi
 if [ "$DB_HOST" != '' ]; then
-    sed -i "s|DB_HOST=.*|DB_HOST=${DB_HOST}|i" /conf/.env
-fi
+    replace_or_insert "DB_HOST" "$DB_HOST"
+ fi
 if [ "$DB_PORT" != '' ]; then
-    sed -i "s|DB_PORT=.*|DB_PORT=${DB_PORT}|i" /conf/.env
-fi
+    replace_or_insert "DB_PORT" "$DB_PORT"
+ fi
 if [ "$DB_DATABASE" != '' ]; then
-    sed -i "s|DB_DATABASE=.*|DB_DATABASE=${DB_DATABASE}|i" /conf/.env
-fi
+    replace_or_insert "DB_DATABASE" "$DB_DATABASE"
+ fi
 if [ "$DB_USERNAME" != '' ]; then
-    sed -i "s|DB_USERNAME=.*|DB_USERNAME=${DB_USERNAME}|i" /conf/.env
-fi
+    replace_or_insert "DB_USERNAME" "$DB_USERNAME"
+ fi
 if [ "$DB_PASSWORD" != '' ]; then
-    sed -i "s|DB_PASSWORD=.*|DB_PASSWORD=${DB_PASSWORD}|i" /conf/.env
-elif [ "$DB_PASSWORD_FILE" != '' ]; then
+    replace_or_insert "DB_PASSWORD" "$DB_PASSWORD"
+ elif [ "$DB_PASSWORD_FILE" != '' ]; then
     value=$(<$DB_PASSWORD_FILE)
-    sed -i "s|DB_PASSWORD=.*|DB_PASSWORD=${value}|i" /conf/.env
-fi
+    replace_or_insert "DB_PASSWORD" "$value"
+ fi
 if [ "$DB_LOG_SQL" != '' ]; then
-    sed -i "s|DB_LOG_SQL=.*|DB_LOG_SQL=${DB_LOG_SQL}|i" /conf/.env
-fi
+    replace_or_insert "DB_LOG_SQL" "$DB_LOG_SQL"
+ fi
 if [ "$DB_LOG_SQL_EXPLAIN" != '' ]; then
-    sed -i "s|DB_LOG_SQL_EXPLAIN=.*|DB_LOG_SQL_EXPLAIN=${DB_LOG_SQL_EXPLAIN}|i" /conf/.env
-fi
+    replace_or_insert "DB_LOG_SQL_EXPLAIN" "$DB_LOG_SQL_EXPLAIN"
+ fi
 if [ "$TIMEZONE" != '' ]; then
-    sed -i "s|TIMEZONE=.*|TIMEZONE=${TIMEZONE}|i" /conf/.env
-fi
+    replace_or_insert "TIMEZONE" "${TIMEZONE//\//\\\/}"
```

Comment:
`${TIMEZONE//\//\\\/}` instead of just `$TIMEZONE` is necessary to escape the `/` in the timezone like `Europe/Berlin`

Reply:
That's why the sed was using `|` before.